cmd/cloudx/client/auth.go

```diff
@@ -37,9 +37,8 @@ func (h *CommandHelper) checkAuthenticated(_ context.Context) error {
 	if c.AccessToken == nil {
 		return ErrNotAuthenticated
 	}
-	if c.AccessToken.Expiry.Before(time.Now()) {
-		return ErrReauthenticate
-	}
+	// TODO should we do some API call here to check whether the token is still valid?
+	// return ErrReauthenticate
```

Comment:
@alnr wdyt? Should we let the token refresh later automatically, or should we do some call that does the refresh at this point?